requirements/requirements-test.txt

```diff
@@ -6,6 +6,7 @@ cython==0.29.24
 garnett==0.7.1
 GitPython==3.1.18
 gsd==2.4.2
+https://gitlab.com/materials-modeling/dynasor/-/archive/1.1/dynasor-1.1.tar.gz
```

Comment:
I have requested a new release of dynasor with a fixed MANIFEST. Once this PR is merged and the new release is made, we can replace this with a PyPI package name / version. https://gitlab.com/materials-modeling/dynasor/-/merge_requests/16